homeassistant/components/nest/__init__.py

```diff
@@ -214,7 +219,7 @@ async def async_config_reload() -> None:
     except SubscriberException as err:
         if DATA_NEST_UNAVAILABLE not in hass.data[DOMAIN]:
             _LOGGER.error("Subscriber error: %s", err)
```

Comment:
It's better to pass the message as an argument to ConfigEntryNotReady. We already log the message as an error and have log spam protection in that exception handler.

Reply:
This used to be the way to do this, then someone refactored to include the new exception handling and left this logging. Removed.